section2/reflection.md

```diff
@@ -2,14 +2,38 @@
+
 3. In this section, we removed some of the supports that you had in Section 1. We didn't give the directions for how to run a file in node and made some directions a bit more vague, intentionally. How did that feel? What was uncomfortable about it? How did it support your learning?
+  * I really liked this removing some of the specific directions. I think it is very beneficial when learning because it makes you think back and use the knowledge you acquired previously. I also think it does a good job of making us repeat things enough that they become habitual.
 
 4. What is a conditional statement? Give one example of a daily life conditional. Give one example of where a conditional is probably used in a web application you use.
 
+  * A conditional statement is how you tell the computer to do either one thing or another. A real life example would be the automatic car windshield wipers. If it is raining the windshield wipers will turn on. If it is not raining the windshield wipers will stay off.
+
 5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do.
 
+  * To add multiple conditions you can use `if` statements. After the initial `if` statement you can use curly brackets '{}' to have the console log a response. Then you can add `else if` with the second condition and the curly brackets to log a response. You can also use the same process for the third condition. You can use else at the end to say if none of the first three conditions are met, this is the response.
+
+  * The computer reads the first condition and if it is met then the computer will stop there and log the response for the first condition. If the condition is not met the computer moves on to the second condition. Again if this condition is met the computer will stop there and log the second conditional response. If the first two conditions are not met the computer will run the third condition. The final condition will be reached if all other conditions prove to be false.
+
+
 6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?
 
-7. What questions do you still have about `if` statements and/or functions?
\ No newline at end of file
+  * The tools available to developers that allows us to check conditions without running the block of code would be the google development tools that you can access through chrome. By typing `command + option + I || J`. This will pull up the chrome developer tools.
+
+
+7. What questions do you still have about `if` statements and/or functions?
+
+  * The main question I still have regarding `if` statements is when you use `else`? Do you have to use `else`? Are there every any conditions that apply to `else` or would those just go under `else if`?
```

Comment:
Usually you will want to put specific conditions under `if` and `else if`, and save `else` as a catch all for any unexpected errors.